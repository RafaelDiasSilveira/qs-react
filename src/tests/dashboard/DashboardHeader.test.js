import React from 'react';
import { shallow } from 'enzyme';

import DashboardHeader from '../../dashboard/DashboardHeader';

test('should render DashboardHeader correctly', () => {
  const wrapper = shallow(<DashboardHeader />);

  expect(wrapper).toMatchSnapshot();
});
