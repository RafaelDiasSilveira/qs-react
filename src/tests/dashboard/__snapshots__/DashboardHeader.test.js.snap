// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render DashboardHeader correctly 1`] = `
<div
  className="text-center"
>
  <h1>
    Minha Estante
  </h1>
</div>
`;
